tests/test_idutils.py

```diff
@@ -51,6 +51,12 @@
     ('doi.org/10.1016/j.epsl.2011.11.037', ['doi', 'handle'],
         '10.1016/j.epsl.2011.11.037',
         'https://doi.org/10.1016/j.epsl.2011.11.037'),
+    ('10.1016/üникóδé-дôΐ', ['doi', 'handle'],
```

Comment:
Interesting case - is this an actual DOI?

Reply:
Based on the spec it could be, though from a brief search (from Zenodo records and the web in general) I didn't find any solid example of such DOIs.